### PR TITLE
Cleanup fs cli warnings

### PR DESCRIPTION
**Pull request by dbw9580**
Cleans up IDE warnings in fs cli.

### CODE REVIEW

File: shell/src/main/java/alluxio/cli/fs/command/PinCommand.java [hunk 1]
```diff
@@ -67,7 +67,7 @@ protected void runPlainPath(AlluxioURI path, CommandLine cl)
     if (!invalidMediumType.isEmpty()) {
       throw new IllegalArgumentException("Invalid medium to pin the file. "
           + String.join(",", invalidMediumType) + " are invalid. "
-          + String.join(",", availableMediumList) + " are valid medium types");
+          + String.join(",", validMediumType) + " are valid medium types");
```
Comment (dbw9580) [inline shell/src/main/java/alluxio/cli/fs/command/PinCommand.java:70]: IDE is warning about `validMediumType` is modified but never queried. This should be a typo.

**Review by maobaolong — APPROVED** [review]
@dbw9580 It really make the code better, thank for your improvement.

**Comment by yyongycy** [comment]
which test fails?

**Review by yyongycy — APPROVED** [review]
LGTM

**Comment by yyongycy** [comment]
alluxio-bot, merge this please!

**Pull request merged**